wger/exercises/templates/muscles/overview.html

```diff
@@ -45,52 +76,54 @@
 <div class="row">
     <div class="col-sm-4 col-xs-6">
         <h4 style="padding-bottom:0.5em;">{% trans "Muscles" %}</h4>
-        <div id="muscle-system"
-             class="muscle-background"
-             style="background-image: url('/static/images/muscles/muscular_system_front.svg');">
+        <div id="muscle-system" class="muscle-background"
+            style="background-image: url('/static/images/muscles/muscular_system_front.svg');">
+            <div id="muscle-systems" class="muscle-background">
+            </div>
+
+
+
         </div>
+
     </div>
     <div class="col-sm-4 col-xs-6">
         {# Why do we need to use |first? #}
         {% regroup muscle_list|first by is_front as muscle_group %}
         {% for group in muscle_group %}
-            {% if group.grouper %}
```

Comment:
Nice job on the formatting here 👍🏾

Reply:
Is it really the best way to format. I think the if statement should be indented considering it is within a for loop